src/account/saga.ts

```diff
@@ -88,7 +88,7 @@ function* initializeAccount() {
   Logger.debug(TAG + '@initializeAccount', 'Creating account')
   try {
     ValoraAnalytics.track(OnboardingEvents.initialize_account_start)
-    yield call(getOrCreateAccount)
+    yield call(getOrCreateCapsuleAccount)
```

Comment:
Ideally, this is broken into two stages.
- [ ] Contract kit initializes the Capsule Wallet
- [ ] Redux saga awaits User Management verification and associates the account to the wallet

Reply:
The generator has the following responsibilities:
* Create an account with Capsule using email and verification code (UserMgmt)
* `yield call(getWallet)` Instantiate the `CapsuleWallet | ReactNativeCapsuleWallet` in memory
* Associate the wallet with the server; 
   * Send biometric-signed public key `secp256r1`.
   * Create an account using CapsuleSDK (e.g. `addAccount(undefined, Function)`)

The steps for creating the user object on the server are independent of associating them to the client.